src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateConversationTest.kt

```diff
@@ -0,0 +1,101 @@
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.bearerToken
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.ErrorResponse
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.assertThat
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.resource.model.conversation.aValidCreateReviewConversationRequest
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.withBody
+
+class CreateConversationTest : IntegrationTestBase() {
+  companion object {
+    private const val URI_TEMPLATE = "/conversations/{prisonNumber}"
+  }
+
+  @Test
+  fun `should return unauthorized given no bearer token`() {
+    webTestClient.post()
+      .uri(URI_TEMPLATE, aValidPrisonNumber())
+      .contentType(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus()
+      .isUnauthorized
+  }
+
+  @Test
+  fun `should return forbidden given bearer token with view only role`() {
+    webTestClient.post()
+      .uri(URI_TEMPLATE, aValidPrisonNumber())
+      .withBody(aValidCreateReviewConversationRequest())
+      .bearerToken(aValidTokenWithViewAuthority(privateKey = keyPair.private))
+      .contentType(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus()
+      .isForbidden
+  }
+
+  @Test
+  fun `should fail to create Conversation given null fields`() {
+    val prisonNumber = aValidPrisonNumber()
+
+    // When
+    val response = webTestClient.post()
+      .uri(URI_TEMPLATE, prisonNumber)
+      .bodyValue(
+        """
+          { }
+        """.trimIndent(),
+      )
+      .bearerToken(aValidTokenWithEditAuthority(privateKey = keyPair.private))
+      .contentType(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus()
+      .isBadRequest
+      .returnResult(ErrorResponse::class.java)
+
+    // Then
+    val actual = response.responseBody.blockFirst()
+    assertThat(actual)
+      .hasStatus(HttpStatus.BAD_REQUEST.value())
+      .hasUserMessageContaining("JSON parse error")
+      .hasUserMessageContaining("value failed for JSON property prisonId due to missing (therefore NULL) value for creator parameter prisonId which is a non-nullable type")
+  }
+
+  @Test
+  fun `should create a new Conversation for prisoner`() {
+    // Given
+    val prisonNumber = aValidPrisonNumber()
+    val createReviewConversationRequest = aValidCreateReviewConversationRequest()
+    val dpsUsername = "auser_gen"
+
+    // When
+    webTestClient.post()
+      .uri(URI_TEMPLATE, prisonNumber)
+      .withBody(createReviewConversationRequest)
+      .bearerToken(
+        aValidTokenWithEditAuthority(
+          username = dpsUsername,
+          privateKey = keyPair.private,
+        ),
+      )
+      .contentType(MediaType.APPLICATION_JSON)
+      .exchange()
+      .expectStatus()
+      .isCreated()
+
+    // Then
+    val conversation = conversationRepository.findAllByPrisonNumber(prisonNumber).first()
```

Comment:
👍 
Nice. We'll probably change this once we have the GET conversation(s) endpoint defined to make the assertion using the GET REST API endpoint rather than repository; but it's OK for now
(we've had weird and difficult problems with some of these tests to do with transaction boundaries; but if it works at the moment than happy days 👍 )

We can also add the assertion that a timeline event was created in a future PR as well, but for the time being this is perfect 👍